tests/data/6-security-schemas/combo-security-def-oneOf-td-valid.jsonld

```diff
@@ -0,0 +1,37 @@
+{
+    "@context": "https://www.w3.org/2022/wot/td/v1.1",
+    "id": "urn:uuid:0804d572-cce8-422a-bb7c-4412fcd56f06",
+    "@type": "Thing",
+    "title": "MyLampThing",
+    "$title": "Combo Security oneOf",
+    "description": "TD for a Lamp Thing",
+    "$description": "TDs can specify a combination of security schemes. To avoid redundancy in this case, e.g. repeating the details of the form elements, a ComboSecurityScheme with oneOf can be used instead.",
```

Comment:
```suggestion
    "$description": "TDs can specify a combination of security schemes. A ComboSecurityScheme with oneOf can be used to avoid possible redundancy, e.g., repetition of the details of the form elements.",
```